src/client/liveblog.ts

```diff
@@ -1,8 +1,38 @@
 // ----- Imports ----- //
 
-import setup from 'client/setup';
 
+import setup from 'client/setup';
+import { fromCapiLiveBlog } from 'item';
+import ReactDOM from 'react-dom';
+import LiveblogBody from 'components/liveblog/body';
+import { createElement as h } from 'react';
+import { Content } from 'mapiThriftModels';
+import { parse } from './parser';
+import { ImageMappings } from 'components/shared/page';
 
 // ----- Run ----- //
 
+declare global {
+    interface Window {
+        __INITIAL__DATA__: Content & {
```

Comment:
Does the full CAPI object get added to the window? Is there anything in a CAPI response that should be private, bearing in mind MAPI has a higher level of access? @davidfurey @alexduf ?

Reply:
We've got a PR to fetch less data in MAPI before sending to apps-rendering. We should remove anything unnecessary there

Reply:
Ok. We can filter out anything that shouldn't be exposed publicly there. In fact, we should do that anyway since that endpoint itself will presumably be public?